application/cms/models.py

```diff
@@ -937,4 +944,78 @@ class LowestLevelOfGeography(db.Model):
     description = db.Column(db.String(255), nullable=True)
     position = db.Column(db.Integer, nullable=False)
 
-    pages = relationship("Page", back_populates="lowest_level_of_geography")
+    pages = db.relationship("Page", back_populates="lowest_level_of_geography")
+
+
+class Topic(db.Model):
+    __tablename__ = "topic"
+
+    id = db.Column(db.Integer, primary_key=True)
+    uri = db.Column(db.String(64), nullable=False)
+    title = db.Column(db.String(255), nullable=False)
+    description = db.Column(db.Text, nullable=True)  # a sentence below topic heading on homepage
+    additional_description = db.Column(db.TEXT, nullable=True)  # short paragraph displayed on topic page
+
+    # Find a list of subtopics that belong to this topic using this relationship
+    subtopics = db.relationship("Subtopic", back_populates="topic")
+
+
+class Subtopic(db.Model):
+    __tablename__ = "subtopic"
+
+    id = db.Column(db.Integer, primary_key=True)
+    uri = db.Column(db.String(64), nullable=False)
+    title = db.Column(db.String(255), nullable=False)
+    position = db.Column(db.Integer, default=0)  # for ordering on the page
+    topic_id = db.Column(db.Integer, ForeignKey("topic.id", name="topic_id_fkey"), nullable=True)
+
+    topic = db.relationship("Topic")
+
+    measures = db.relationship(
+        "Measure",
+        lazy="subquery",
+        secondary=subtopic_measure,
+        primaryjoin="Subtopic.id == subtopic_measure.columns.subtopic_id",
+        secondaryjoin="Measure.id == subtopic_measure.columns.measure_id",
+        back_populates="subtopics",
+    )
```

Comment:
I think this can be a lot simpler - you should just be able to do `db.relationship("Measure", secondary="subtopic_measure", backref="subtopics", order_by=something)`

Reply:
(Arguments about backref vs back_populates aside)